### DISCUSSION TITLE
How to remove grid on line chart?

### DISCUSSION THREAD

**Discussion by nithinreddyy** [atoti/atoti#471] [Q&A] [answered]
I tried plotting the line charts with dates. Below is the plot

![Capture](https://user-images.githubusercontent.com/56256685/134647307-0a8dd06c-5bf2-4539-9f0d-f4b38bbf0d22.PNG)

Can we remove those black lines behind the curve and change the dates to month format?

**Comment by bloussou** [comment, marked as answer]
Hello,
To remove the grid you can add the following plotly configuration in your cell metadata :
```json
"plotly": {
                "layout": {
                    "xaxis": {
                        "showgrid": false
                    },
                    "yaxis": {
                        "showgrid": false
                    }
                }
            }
```
![image](https://user-images.githubusercontent.com/32448554/151340397-e0759a19-0043-4e83-8e81-f54539941e42.png)
Without the config :
![image](https://user-images.githubusercontent.com/32448554/151340496-c2d16742-2006-4bff-8936-bb22345ee06d.png)
Related to https://github.com/atoti/atoti/issues/384
